public/css/horizontal_menu.css

```diff
@@ -94,11 +94,11 @@
 }
 
 .horizontal-menu.full-width li:first-child {
-  text-align: left;
+/*  text-align: left;*/
```

Comment:
Seems like a duplicate. But also, since we're using git, just delete stuff instead of commenting it out. It's easy to get it back from version control :)